README.md

```diff
@@ -1,29 +1,39 @@
 llvm-cbe
 ========
 
-resurrected LLVM "C Backend", with improvements
+Resurrected LLVM C backend, with improvements.
 
-
-INSTALLATION INSTRUCTIONS
+Installation instructions
 =========================
 
-This version of the LLVM-CBE library works with LLVM 8.0. You will have
-to compile this version of LLVM before you try to use LLVM-CBE. This
-guide will walk you through the compilation and installation of both
-tools and show usage statements to verify that the LLVM-CBE library is
-compiled correctly.
+This version of the LLVM C backend works with LLVM 10.0.
 
-The library is known to compile on various Linux versions (Redhat,
-Mageia, Ubuntu, Debian), Mac OS X, and Windows (Mingw-w64).
+Earlier versions are supported too (LLVM 7 and earlier should work).
 
 Step 1: Installing LLVM
 =======================
 
-LLVM-CBE relies on specific LLVM internals, and so it is best to use
-it with a specific revision of the LLVM development tree. Currently,
-llvm-cbe works with the LLVM 8.0 release version and autotools.
+Either install the LLVM packages on your system:
+--------------------------------------------
 
-Note: to convert C to LLVM IR to run the tests, you will also need a C compiler such as clang.
+On macOS, use [pkgsrc](http://pkgsrc.joyent.com/install-on-osx/) and run the following commands:
```

Comment:
Please add Windows too. That's why I added my previous review.

Reply:
Ok, note that I didn't and won't be able to try for now on Windows though...

Reply:
I can test that. I am actually on Windows, and I have Ubuntu inside it in my WSL.

Reply:
For WSL, the Linux instructions work by design. For native Windows, it's another story...

Reply:
@aminya Note that I connected to my work Windows machine now and will see there

Reply:
Answer: prebuilt packages from LLVM.org for Windows are built statically, but with the LLVM-C .lib and .dll available too. No support for the LLVM C++ API.

Aka, feasible only with building everything for now.

Reply:
I can find a way for that. Stay tuned.